src/registries/npm-registry.js

```diff
@@ -65,7 +65,7 @@ function normalizePath(val: mixed): ?string {
   }
 
   if (typeof val !== 'string') {
-    val = '' + (val: any);
+    val = String(val);
```

Comment:
Thanks for taking care of a fix! Didn't find time at the beginning of the week to dig deeper!

One question. Does it make sense to have this at all? In the end, this will go to `path.resolve`, maybe it makes sense to throw an error instead?